src/Microsoft.VisualBasic/tests/VB/DriveInfo.Windows.Tests.vb

```diff
@@ -0,0 +1,174 @@
+            Dim VolumeName As StringBuilder = New StringBuilder(VolNameLen)
+            Const FileSystemNameLen As Integer = 50
+            Dim FileSystemName As StringBuilder = New StringBuilder(FileSystemNameLen)
+            Dim serialNumber, maxFileNameLen, fileSystemFlags As Integer
+            Dim r As Boolean = GetVolumeInformation(validDrive.Name, VolumeName, VolNameLen, serialNumber, maxFileNameLen, fileSystemFlags, FileSystemName, FileSystemNameLen)
+            Dim FileSystemType As String = FileSystemName.ToString
+
+            If r Then
+                Assert.Equal(FileSystemType, validDrive.DriveFormat)
+            Else
+                Assert.Throws(Of IO.IOException)(Function() validDrive.DriveFormat)
+            End If
+
+        End Sub
+
+        <Fact>
+        <PlatformSpecific(TestPlatforms.Windows)>
+        Public Shared Sub TestDriveType()
+            Dim validDrive As IO.DriveInfo = FileSystem.Drives().Where(Function(d) d.DriveType = IO.DriveType.Fixed).First()
+            Dim expectedDriveType As Integer = GetDriveType(validDrive.Name)
+            Assert.Equal(CType(expectedDriveType, IO.DriveType), validDrive.DriveType)
+        End Sub
+
+        <Fact>
+        <PlatformSpecific(TestPlatforms.Windows)>
+        <SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "GetDiskFreeSpaceEx blocked in AC")>
+        Public Shared Sub TestValidDiskSpaceProperties()
+            Dim win32Result As Boolean
+            Dim fbUser As Long = -1
+            Dim tbUser As Long
+            Dim fbTotal As Long
+            Dim drive As IO.DriveInfo
+            drive = FileSystem.Drives().Where(Function(d) d.DriveType = IO.DriveType.Fixed).First()
+            If drive.IsReady Then
+                win32Result = GetDiskFreeSpaceEx(drive.Name, fbUser, tbUser, fbTotal)
+                Assert.[True](win32Result)
+
+                If fbUser <> drive.AvailableFreeSpace Then
+                    Assert.[True](drive.AvailableFreeSpace >= 0)
+                End If
+
+                ' valid property getters shouldn't throw
+                Dim name As String = drive.Name
+                Dim format As String = drive.DriveFormat
+                Assert.Equal(name, drive.ToString())
+
+                ' totalsize should not change for a fixed drive.
+                Assert.Equal(tbUser, drive.TotalSize)
+
+                If fbTotal <> drive.TotalFreeSpace Then
+                    Assert.[True](drive.TotalFreeSpace >= 0)
+                End If
+            End If
+        End Sub
+
+        <Fact>
+        <PlatformSpecific(TestPlatforms.Windows)>
+        Public Shared Sub GetVolumeLabel_Returns_CorrectLabel()
+            If PlatformDetection.IsInAppContainer Then
+                Assert.Throws(Of UnauthorizedAccessException)(Sub() DoDriveCheck())
+            Else
+                DoDriveCheck()
+            End If
+        End Sub
+
+
+        <DllImport("kernel32.dll", SetLastError:=True)>
```

Comment:
move these interops into a helpers.vb file

Reply:
Not worth it if we're going to delete them anyway to make Unix work. In this case, `Directory.GetLogicalDrives()` can be used instead.